pkg/pillar/cmd/zedagent/parseconfig_test.go

```diff
@@ -1254,3 +1264,73 @@ func TestParseSRIOV(t *testing.T) {
 		},
 	}))
 }
+
+func TestParsePatchEnvelope(t *testing.T) {
+	g := NewGomegaWithT(t)
+	getconfigCtx := initGetConfigCtx(g)
+
```

Comment:
```suggestion
t.Parallel()
```
I am wondering why yetus is not complaining ...

Is gomega doing this under the hood?

Reply:
AFAIK gomega is not testing framework, ginko is. So I'm not sure it's doing this under the hood...